eng/pipelines/aggregate-reports.yml

```diff
@@ -103,10 +103,21 @@ jobs:
         publishJUnitResults: false
         goals: 'clean package exec:java'
 
+    - task: UsePythonVersion@0
```

Comment:
Why was this necessary? I don't see any usages of python.

Reply:
Gah! I'm not even sure how I could have copy/paste error'd that. I'll remove it with the next PR.